test/cypress/tests/utils/popover.cy.ts

```diff
@@ -173,8 +182,9 @@ describe('Popover', () => {
   });
 
   it('should perform radiobutton-like behavior among the items that have toggle property value set to the same string value', () => {
-    const items: PopoverItem[] = [
+    const items = [
       {
+        type: 'default',
```

Comment:
Maybe use enum for popover type

Reply:
Not sure if it is critical. TypeScript will not allow you to enter anything except 'default' and 'separator' anyway